examples/python/Makefile

```diff
@@ -66,7 +66,7 @@ DEFINES += -D_XOPEN_SOURCE=500 -D_BSD_SOURCE
 CDEFINES   += $(DEFINES)
 CXXDEFINES += $(DEFINES)
 
-FLAGS     = -g -Wall $(shell python3-config --cflags) -O1 
+FLAGS     = -g -Wall $(shell python3-config --cflags) -O1 -fPIC
```

Comment:
I was compiling with Verilator. I'm unsure if we need this for not Verilator, so we may need some conditional logic here.

Reply:
I don't think it breaks RHEL/Centos, but good to check.